packages/source-iottwinmaker/src/knowledgeGraph-module/KGDataModule.ts

```diff
@@ -0,0 +1,44 @@
+import {
+  ExecuteQueryCommand,
+  ExecuteQueryCommandOutput,
+  ExecuteQueryCommandInput,
+  IoTTwinMakerClient,
+  ColumnDescription,
+  Row,
+} from '@aws-sdk/client-iottwinmaker';
+import { KGDataModuleInput } from './types';
+import { TwinMakerKGQueryDataModule } from '../types';
+
+export class KGDataModule implements TwinMakerKGQueryDataModule {
+  private twinMakerClient: IoTTwinMakerClient;
+  private workspaceId: string;
+
+  constructor(input: KGDataModuleInput) {
+    this.twinMakerClient = input.twinMakerClient;
+    this.workspaceId = input.workspaceId;
+  }
+
+  executeQuery = async (queryStatement: string, maxResultsCount?: number): Promise<ExecuteQueryCommandOutput> => {
```

Comment:
should `maxResultsCount` be `maxPagesCount` instead? otherwise we should update the code to not return more than `maxResultsCount` number of rows

Reply:
addressed.